applications/sasquatch-backpack/templates/topics.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.strimzi.enabled }}
```

Comment:
You can't really test this effectively here, so just assume Strimzi is deployed that that backpack depends on kafka being deployed.